modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/entity/BaseEpisode.kt

```diff
@@ -41,6 +41,7 @@ sealed interface BaseEpisode {
     var playedUpToModified: Long?
     var playErrorDetails: String?
     var downloadErrorDetails: String?
+    var deselectedChapters: String?
```

Comment:
Using `String` for keeping deselected chapter indices is a bit troublesome. Now, whenever we want to access this we have to know internal mapping using the comma, and we have to keep in check data integrity because we can put whatever in a `String`.

I'd like to propose a simple class like this one.

```kotlin
class ChapterIndices(delegate: List<Int>) : List<Int> by delegate
```

This way we can write a single mapper for `Room` to understand how to store this data and outside of it we can use it as a list and have access to `Int` representing indices.

Reply:
I converted the return type to `ChapterIndices` in https://github.com/Automattic/pocket-casts-android/pull/1856/commits/a2a8f1b5b8608dce4666e525436f45007a460bdd and https://github.com/Automattic/pocket-casts-android/pull/1856/commits/4b41ae5872d29caddb45ea84f38a3f4cce481f2b. It now looks more type-safe 👍